runtime/polkadot/src/lib.rs

```diff
@@ -328,11 +329,23 @@ impl pallet_session::historical::Config for Runtime {
 }
 
 parameter_types! {
-	// no signed phase for now, just unsigned.
-	pub const SignedPhase: u32 = 0;
+	// phase durations. 1/4 of the last session for each.
+	pub const SignedPhase: u32 = EPOCH_DURATION_IN_SLOTS / 4;
 	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_SLOTS / 4;
 
-	// fallback: run election on-chain.
+	// signed config
+	pub const SignedMaxSubmissions: u32 = 16;
+	pub const SignedDepositBase: Balance = deposit(1, 0);
+	// A typical solution currently occupies 120kb. This formula is currently adjusted such that a
+	// typical solution will spend approximately equal amounts on the base and per-byte deposits.
+	pub const SignedDepositByte: Balance = deposit(1, 0) / (120 * 1024 * 1024);
```

Comment:
isn't this one too many 1024s?

Reply:
I like the idea, basically *`deposit(1, 0)` per 120kb*

Reply:
and since the 120kb is changing all the time, they could be a bit more fixed, like `deposit(1, 0) per 50kb` for both chains. Note that `fn deposit` of kusama is already adjusted for the lower entry barrier.

Reply:
1. Good spot! Yeah, I was thinking in mb for some reason.
2. Adjusted all the chains' rates to a base deposit per 50kb as suggested: https://github.com/paritytech/polkadot/pull/2793/commits/f5b6dd5d66d95fab425370702c3e296565641fda.